schemas/component/content.json

```diff
@@ -17,6 +17,63 @@
       "type": "string",
       "content": true,
       "multiline": true
+    },
+    "conditionals": {
```

Comment:
Is the schema for conditionals re-usable from branching? If so do we have those we can reference, if not, part of me wants to call them something like content_visibility_conditions or something just to be explicit